daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -598,6 +598,37 @@ def _close(self):
         self.key_manager.close()
 
 
+class RazerHuntsmanV3ProTKL(_RazerDeviceBrightnessSuspend):
```

Comment:
Can you add the comment `# TODO Should become _RippleKeyboard once kernel support for driver mode is implemented` above, since this is another Analog keyboard that (probably) cannot be used in driver mode currently.